### ISSUE TITLE
LOCA Heading to define which GEOL model was used

### ISSUE THREAD

**Issue by SolidRockProgrammer** [australiangeomechanics/agsformat#72]
I’m ok with historic data partially using GEOL extended, as long as we document that situation is the case in a field on LOCA.  I think adding a heading to GEOL, hence populating it for every record is not needed, but I could accept a GEOL heading if good reasons are defined.  Have a PA: 
E – full use of extended component headings, such that a full description could be created from the data
P – Partial use of extended component headings, only acceptable for historic data
S – Simple headings used

**Comment by digitalgeotechnics** [comment]
**edit - issue not seeded, but similar idea discussed in...
Issue seeded from https://github.com/australiangeomechanics/agsformat/discussions/71#discussion-4998788 by @RobDayARUP

> A new boolean Y/N field GEOL_ISCD needs to be added to GEOL to indicate whether the data in GEOL_DESC is a non-component description or can be COMPLETELY defined by the component description. An ‘N’ would indicate it is a non-component description and any component fields are for supplementary information.

**Comment by SolidRockProgrammer** [comment]
It wasn't seeded by that, we discussed it 2 months ago

Sent by mobile - Phil Wade
________________________________

**Comment by digitalgeotechnics** [comment]
similar concept though?

**Comment by digitalgeotechnics** [comment]
The purpose of the indicator field needs to be considered.  Is it (1) to inform any algorithmic tools parsing the description of how to handle the data in GEOL? Or is it (2) for "contractual reliability" - to indicate which is the superior data set in the presence of a conflict between `GEOL_DESC` and the extendedGEOL fields?

**Comment by SolidRockProgrammer** [comment]
both (1) and (2), if "P" one can't refer to the extendedGEOL for all data.

**Comment by digitalgeotechnics** [comment]
Such a field could be placed at several level in the AGS hierarchy - `PROJ` ("every LOCA in this file follows the stated GEOL model"), `LOCA` (this test locations GEOL table uses the stated model"), `GEOL` ("this single GEOL description is intended to use this model").  

In the case where historic drilling information is being digitised from `GEOL_DESC` into the component fields, it's plausible (probable?) that there might only be translation of part of the depth of the hole (eg only from surface to the relevant depth, or only within a certain range above/below a tunnel obvert/invert).  In this case only the `GEOL` level field would be adequate

**Comment by SolidRockProgrammer** [comment]
@StephenDurham @AGS-Nat @SThorin 
Please enter the conversation

**Comment by digitalgeotechnics** [comment]
I recall when the working group discussed this we discussed in the context of considered the existing guidance note in `GEOL`

> If component description headings are input, they shall take precedence over the combined GEOL_DESC

With this clause then such a field could be considered redundant because:
in case (1) the same can be achieved by a test that `GEOL_1SN` or `GEOL_1RNM` are not an empty string (although perhaps a guidance note might be required to say that at least one of these fields is required if the extended model is used). 
in case (2), I'm not sure that there is a requirement to specify this because the clause could be read as saying "if the data is in `GEOL_DESC` but not found in a component field it effectively doesn't exist for contractual purposes"

**Comment by RobDayArup** [comment]
I think classifying descriptions as simple/partial/extended at a LOCA level is too coarse.  It needs to be at the individual GEOL data entry level.  See https://github.com/australiangeomechanics/agsformat/discussions/71#discussion-4998788

**Comment by SolidRockProgrammer** [comment]
I don't consider the words on https://australiangeomechanics.org/ags-4-1-au/ to be part of the definition of AGS 4.1 AU.  Is that everyone's understanding?

**Comment by RobDayArup** [comment]
I don't know what the committee think, but for my part I can say I have been making decisions for the last year on the basis that I can rely on it.

**Comment by RobDayArup** [comment]
... and for the reasons I have discussed it makes more sense than a blanket all or nothing approach on GEOL.

**Comment by digitalgeotechnics** [comment]
> I don't consider the words on https://australiangeomechanics.org/ags-4-1-au/ to be part of the definition of AGS 4.1 AU. Is that everyone's understanding?

yeah, i would have thought of that as an introduction or more marketing-esc.  there is no formal tracking of revision history on that page either that i know of
